apps/portal/src/app/features/file-processor/file-processor.component.ts

```diff
@@ -145,15 +145,19 @@ export class FileProcessorComponent implements OnInit, OnDestroy {
     );
   }
 
-  getSeverity(status: number): string {
-    const severityMap: { [key: number]: string } = {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  getSeverity(status: number): any {
+    const severityMap: {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      [key: number]: any;
+    } = {
       1: 'warning',
       2: 'primary',
       3: 'info',
       4: 'success',
       5: 'danger',
     };
-    return severityMap[status] || 'error';
+    return severityMap[status] || undefined;
```

Comment:
Should this be undefined now? Maybe when undefined it will set it as primary? Can you confirm this once?

Reply:
Yes when undefined it seems to set as primary

undefined
![image](https://github.com/user-attachments/assets/00004ff5-c2ee-4ce9-a1f4-5f8ab84291ba)

error
![image](https://github.com/user-attachments/assets/996fba76-e808-44b0-b50a-518989b24b29)

primary
![image](https://github.com/user-attachments/assets/1e7255bd-9b9a-413c-95e9-3bd7d44ff1ef)

Reply:
@kshitij-k-osmosys  I did not get, why are we keeping it as undefined? Do we want to set as Primary By default for cases which are not handled? Is Primary not being used for any other case (QUEUED in this case)? So defualt should be something like Secondary so that any status which is not handled will have color defined by secondary?

Reply:
Update unhandled cases to use `secondary`